packages/debug/src/browser/debug-session-connection.ts

```diff
@@ -243,16 +243,22 @@ export class DebugSessionConnection implements Disposable {
         const connection = await this.connectionPromise;
         const messageStr = JSON.stringify(message);
         if (this.traceOutputChannel) {
-            this.traceOutputChannel.appendLine(`${this.sessionId.substring(0, 8)} theia -> adapter: ${messageStr}`);
+            // keep getMilliseconds
+            const now = new Date();
+            const dateStr = `${now.toLocaleString(undefined, { hour12: false })}.${now.getMilliseconds()}`;
+            this.traceOutputChannel.appendLine(`${this.sessionId.substring(0, 8)} ${dateStr} theia -> adapter: ${JSON.stringify(message, null, 4)}`);
```

Comment:
@DanboDuan if you don't need `null` explicitly, please use `undefined` instead, it fails the build unless the line is suppressed:

```suggestion
            this.traceOutputChannel.appendLine(`${this.sessionId.substring(0, 8)} ${dateStr} theia -> adapter: ${JSON.stringify(message, undefined, 4)}`);
```

The same is similar for `handleMessage`.

Reply:
Thx, done.